deployment/builder/app/main.py

```diff
@@ -36,15 +36,25 @@
 app = Flask(__name__)
 
 
+class BuildError(Exception):
+    """Couldn't build/update our images."""
+
+
 @app.route("/", methods=["get", "post"])
 def main():
     data = request.json or {}
     repo = data.get("repository", {}).get("full_name", "")
     branch = data.get("ref", "").replace("refs/heads/", "")
     branch_config = REPOS.get(repo)
     if branch_config and branch in branch_config:
-        res = build(branch, branch_config["names"], **branch_config[branch])
-        if res:
+        try:
+            res = update(branch, branch_config["names"], **branch_config[branch])
+        except BuildError:
+            print("Error building", file=sys.stderr)
```

Comment:
There's `_print` for printing to stderr. That's not the best function name, but we should use the same way to print to stderr in the whole file.